packages/common/src/labelToResource.ts

```diff
@@ -0,0 +1,20 @@
+import { ROOT_NAMESPACE } from "./resourceToLabel";
+
+export function labelToResource(label: string): {
```

Comment:
fwiw I don't think we should encourage this behavior of going from a string label to a resource

we should probably just keep this as an internal util close to where we need it (i.e. worldgen)

Reply:
Okay that makes sense! I think we need it in a few places lemme see